### PR TITLE
Change the swagger to what is actually implemented

### PR DESCRIPTION
**Pull request by tmuehl**
Adopt to the swagger file what is actually implemented in the python version

@valgog @v-stepanov @jkliff @CyberDem0n

### CODE REVIEW

**Comment by review-ninja** [comment]
<a href="https://app.review.ninja/zalando/nakadi/pull/41" target="_blank">ReviewNinja

File: nakadi/swagger.yaml [hunk 1]
```diff
@@ -227,6 +216,10 @@ paths:
         - name: event
           in: body
           description: Even being sent.
+          schema:
```
Comment (v-stepanov) [inline nakadi/swagger.yaml]: This will make connexion validate the request body and it will fail as it is not a JSON

**Comment by v-stepanov** [comment]
Maybe we can just have some thumbsup as a approval? Ninja for just approvals is a big overhead

**Comment by v-stepanov** [comment]
:+1:

**Comment by v-stepanov** [comment]
hmm, it seems that thumbsup automatically adds a ninja star

**Comment by tmuehl** [comment]
Yea, that would be great

**Pull request merged**